addons/mail/static/src/new/web/activity/activity.js

```diff
@@ -12,6 +12,7 @@ import { computeDelay } from "@mail/new/utils/dates";
 import { useAttachmentUploader } from "@mail/new/attachments/attachment_uploader_hook";
 
 import { _t } from "@web/core/l10n/translation";
+import { useMessaging } from "@mail/new/core/messaging_hook";
```

Comment:
All other `import`s of the file are grouped by module. Only these two don't follow the same logic, which is a bit odd.